Epona/sources/Debug.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+* Copyright (C) 2018 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+#include <Epona/Debug.hpp>
+
+namespace epona
+{
+namespace debug
+{
+
+void Debug::QuickhullConvexHullCall(
+        QuickhullConvexHull<std::vector<glm::dvec3>>& convexHull,
+        std::vector<glm::dvec3>& vertexBuffer
+    )
+#ifdef EPONA_DEBUG
+{
+    if (quickhullConvexHullCallback)
+    {
+        quickhullConvexHullCallback(convexHull, vertexBuffer);
+    }
+}
+#else
+{
+}
+#endif
+
+std::function<
+    void(QuickhullConvexHull<std::vector<glm::dvec3>>&, std::vector<glm::dvec3>&)
+> Debug::quickhullConvexHullCallback = nullptr;
```

Comment:
`nullptr` could be substituted with dummy lambda to avoid unnecessary branching

Reply:
How would that cause a branching?

Reply:
Right now you have `if (quickhullConvexHullCallback)` which is only the case when nothing was set there

Reply:
I see.